paddle/fluid/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 add_subdirectory(memory)
 add_subdirectory(platform)
 add_subdirectory(framework)
+#add_subdirectory(distributed)
```

Comment:
why comment on this directory?

Reply:
because we don't use this, had delete this.